### ISSUE TITLE
toggle window history

### ISSUE THREAD

**Issue by aloispichler** [hyprwm/Hyprland#3890]
### Description

I have seen
```
bind= ALT, Tab, workspace, m+1        # next workspace
bind= ALT Shift, Tab, workspace, m-1  # previous workspace
```
recently, and this works nicely.
And it would be nice, however, to have
```
bind= ALT, Tab, workspace, h-1         # back in history
bind= ALT Shift, Tab, workspace, h+1   # forward
```
to toggle the history of workspaces back- and forth, or
```
bind= ALT, Tab, window, h+1            # window history
bind= ALT Shift, Tab, window, h-1
```
for windows. 
Why is it important? Because `alt+tab` can be comfortably run with one, the _left_ hand only. This is often better than regular window or workspace switching keybinds.

**Comment by thejch** [comment]
not sure if there's a forward, but there's going back in history: `workspace, previous`

**Comment by aloispichler** [comment]
Yes, true; there is even `focuscurrentorlast`, which is often better.
But they all have limited functionality. Unfortunately, they discard and ignore the history except the very last workspace/ window.

**Comment by aloispichler** [comment]
[hyprswitch](https://github.com/H3rmt/hyprswitch) solves the `Alt-Tab` problem.

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
